printer_mods/Hobson318/Belt_Tension_Wrench/readme.md

```diff
@@ -0,0 +1,5 @@
+
+# Tools
+This is a wrench that goes over the AB belt tensioner nuts to help with fine tuning the belt tension.
+![IMG_1916 (Medium)](https://user-images.githubusercontent.com/85077660/152629218-7e89bd77-3347-429a-a88f-4fd38c0af301.JPG)
+![IMG_1917 (Medium)](https://user-images.githubusercontent.com/85077660/152629220-cd28dbe7-2319-42d4-ac1c-7691034d256d.JPG)
```

Comment:
Same for these. replace them with relative links so they always track with the current state of the repo.

Reply:
I would but i don’t know how. And I’m not having a good time trying to find something that actually explains what to do. Just stuff that tells you it’s available and assumes you know how to make it work. If you can eli5 how to do what you want that would be great. If not and you just want to trash the pull request that’s fine to. Sorry for any inconvenience this is causing.

Reply:
sure.
1. Add an "Images" folder to the mod
2. place the images in the folder
3. Link to them using `(./Images/<image_name>)`